<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Toolbar</class>
 <widget class="QWidget" name="Toolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>94</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999999</width>
    <height>9999999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Toolbar - Spire</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="m_headerLayout">
     <item>
      <layout class="QHBoxLayout" name="m_windowManagerLayout">
       <item>
        <widget class="QLabel" name="m_windowManagerLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Window Manager</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_windowManagerButton">
         <property name="styleSheet">
          <string notr="true">QToolButton::menu-indicator {
  width: 0px;
  height: 0px;
}
</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="m_headerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="m_recentlyClosedLayout">
       <item>
        <widget class="QLabel" name="m_recentlyClosedLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Recently Closed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_recentlyClosedButton">
         <property name="styleSheet">
          <string notr="true">QToolButton::menu-indicator {
  width: 0px;
  height: 0px;
}
</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_componentsLayout">
     <item>
      <widget class="QGroupBox" name="m_profileGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>106</red>
             <green>104</green>
             <blue>100</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Profile</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QToolButton" name="m_newCanvasButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>New Canvas</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/table2.png</normaloff>:/icons/table2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_profileButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>View Profile</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_keyBindingsButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Key Bindings</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton::menu-indicator {
  width: 0px;
  height: 0px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/keyboard_key.png</normaloff>:/icons/keyboard_key.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_logoutButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Logout</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_dataManagementGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="title">
        <string>Data Management</string>
       </property>
       <layout class="QHBoxLayout" name="m_marketDataLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QToolButton" name="m_openBookViewButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Book View</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/book_blue.png</normaloff>:/icons/book_blue.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_openTimeAndSalesButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Time and Sales</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/time_and_sales.png</normaloff>:/icons/time_and_sales.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_openChartButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Chart Window</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/chart_line.png</normaloff>:/icons/chart_line.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_openDashboardButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Dashboard Window</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/code_colored.png</normaloff>:/icons/code_colored.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_openOrderImbalanceIndicatorButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Open Order Imbalance Indicator</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/scales.png</normaloff>:/icons/scales.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_openBlotterButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Open Blotter</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton::menu-indicator {
  width: 0px;
  height: 0px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/window_earth.png</normaloff>:/icons/window_earth.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_administrationGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Admin</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QToolButton" name="m_accountsButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>View Accounts</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/application_server.png</normaloff>:/icons/application_server.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_openPortfolioViewerButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>View Portfolios</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../LegacyResources/LegacyResources.qrc">
            <normaloff>:/icons/briefcase.png</normaloff>:/icons/briefcase.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../LegacyResources/LegacyResources.qrc"/>
 </resources>
 <connections/>
</ui>
